packages/main/src/List.ts

```diff
@@ -177,8 +177,12 @@ type ListMoveEventDetail = MoveEventDetail;
 	],
 })
 /**
- * Fired when an item is activated, unless the item's `type` property
+ * Fired when an item is clicked, unless the item's `type` property
```

Comment:
`activated` was the correct term as this can be also done via the keyboard.

In UI5 the term `click` is broadly used and signifies not only mouse/touch, but also keyboard interaction.

For example, with `ui5-button` there is only a `click` event, which however triggers on both mouse/keyboard.